<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exporting_variables</class>
 <widget class="QWidget" name="exporting_variables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>338</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>205</width>
    <height>338</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>205</width>
    <height>338</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <widget class="QPlainTextEdit" name="single_variables_to_shelve">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="placeholderText">
    <string notr="true">Variables to export. Comma seperated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Variables to Shelve:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="folder_location">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_variables_push_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File Name(*db)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UI_ShelvedVariables</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="failure_text_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>Failure Text Label</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
